src/clarity-angular/data/datagrid/datagrid.spec.ts

```diff
@@ -107,6 +107,21 @@ export default function(): void {
                     expect(context.testComponent.nbRefreshed).toBe(1);
                 });
 
+                it("emits twice when the filters change when currentPage > 1", function() {
```

Comment:
I'm surprised because I don't remember this being what we agreed on. Blocking the PR until we discuss it offline.